.github/workflows/test_sanity_check.yml

```diff
@@ -74,6 +78,7 @@ jobs:
       - name: Run setup test environment workflow
         uses: './.github/actions/setup_test_environment'
         with:
+          ARTIFACT_GROUP: ${{ inputs.artifact_group }}
           ARTIFACT_RUN_ID: ${{ env.ARTIFACT_RUN_ID }}
           AMDGPU_FAMILIES: ${{ inputs.amdgpu_families }}
```

Comment:
Ah, we should have also removed `AMDGPU_FAMILIES` here, given the rename to `ARTIFACT_GROUP`. I see warnings for this now:

https://github.com/ROCm/TheRock/actions/runs/18935972955/job/54069345939#annotation:6:4

> [Run setup test environment workflow](https://github.com/ROCm/TheRock/actions/runs/18935972955/job/54069345939#annotation:6:4)
Unexpected input(s) 'AMDGPU_FAMILIES', valid inputs are ['VENV_DIR', 'OUTPUT_ARTIFACTS_DIR', 'ARTIFACT_GROUP', 'ARTIFACT_RUN_ID', 'FETCH_ARTIFACT_ARGS', 'IS_PR_FROM_FORK']

cc @geomin12